flink-end-to-end-tests/flink-end-to-end-tests-aws-kinesis-streams/pom.xml

```diff
@@ -65,6 +65,13 @@
 			<artifactId>jackson-databind</artifactId>
 			<scope>test</scope>
 		</dependency>
+
+		<dependency>
+			<groupId>org.apache.flink</groupId>
+			<artifactId>flink-connector-aws-base</artifactId>
```

Comment:
same question as for firehose. Since they don't use the connector directly (but add the sql-connector jar to the jobs classpath) it shouldn't be needed.